Problem with building using webpack

Line https://github.com/fails-components/webtransport/blob/b2e57e104990cc2db85d0fd19426e8b919217ac0/main/lib/webtransport.browser.js#L6

`.js` extension is not passed. Please add.

Without it I got following error

```
ERROR in ./node_modules/@fails-components/webtransport/lib/webtransport.browser.js 6:0-43
Module not found: Error: Can't resolve './error' in '/var/data/node_modules/@fails-components/webtransport/lib'
Did you mean 'error.js'?
BREAKING CHANGE: The request './error' failed to resolve only because it was resolved as fully specified
```

--- a/main/eslint.config.js
+++ b/main/eslint.config.js
@@ -1,6 +1,8 @@
 import js from '@eslint/js'
 import promise from 'eslint-plugin-promise'
 import babelParser from '@babel/eslint-parser'
+import importplug from 'eslint-plugin-import'
+// eslint-disable-next-line import/extensions
 import eslintPluginPrettierRecommended from 'eslint-plugin-prettier/recommended'
 import globals from 'globals'
 
@@ -9,7 +11,8 @@ export default [
   eslintPluginPrettierRecommended,
   {
     plugins: {
-      promise
+      promise,
+      import: importplug
     },
     languageOptions: {
       globals: {
@@ -37,7 +40,15 @@ export default [
       'import/export': 0,
       'promise/catch-or-return': 'error',
       'no-useless-return': 1,
-      camelcase: 1
+      camelcase: 1,
+      'import/extensions': [
+        'error',
+        'always',
+        {
+          js: 'always',
+          jsx: 'always'
+        }
+      ]
     }
   },
   {

--- a/main/lib/webtransport.browser.js
+++ b/main/lib/webtransport.browser.js
@@ -3,7 +3,7 @@ import { HttpWTSession } from './session.js'
 import { HttpClient } from './client.js'
 import { Http2WebTransportBrowser } from './http2/browser/browser.js'
 import { logger } from './utils.js'
-import { WebTransportError } from './error'
+import { WebTransportError } from './error.js'
 
 const log = logger(`webtransport:browser()`)
 

--- a/main/package.json
+++ b/main/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@fails-components/webtransport",
-  "version": "1.3.0",
+  "version": "1.3.1",
   "description": "A component to add webtransport support (server and client) to node.js using libquiche",
   "exports": {
     ".": {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@fails-components/webtransport-workspace",
-  "version": "1.3.0",
+  "version": "1.3.1",
   "description": "A component to add webtransport support (server and client) to node.js using libquiche",
   "repository": {
     "type": "git",

--- a/test/eslint.config.js
+++ b/test/eslint.config.js
@@ -1,6 +1,8 @@
 import js from '@eslint/js'
 import promise from 'eslint-plugin-promise'
+import importplug from 'eslint-plugin-import'
 import babelParser from '@babel/eslint-parser'
+// eslint-disable-next-line import/extensions
 import eslintPluginPrettierRecommended from 'eslint-plugin-prettier/recommended'
 import globals from 'globals'
 
@@ -9,7 +11,8 @@ export default [
   eslintPluginPrettierRecommended,
   {
     plugins: {
-      promise
+      promise,
+      import: importplug
     },
     languageOptions: {
       globals: {
@@ -37,7 +40,15 @@ export default [
       'import/export': 0,
       'promise/catch-or-return': 'error',
       'no-useless-return': 1,
-      camelcase: 1
+      camelcase: 1,
+      'import/extensions': [
+        'error',
+        'always',
+        {
+          js: 'always',
+          jsx: 'always'
+        }
+      ]
     }
   },
   {

--- a/test/package.json
+++ b/test/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@fails-components/webtransport-tests",
-  "version": "1.3.0",
+  "version": "1.3.1",
   "description": "An automated test for the webtransport component",
   "exports": {
     ".": {

--- a/transports/http3-quiche/eslint.config.js
+++ b/transports/http3-quiche/eslint.config.js
@@ -1,21 +1,22 @@
-
-import js from "@eslint/js"
-import promise from "eslint-plugin-promise"
-import babelParser from "@babel/eslint-parser"
+import js from '@eslint/js'
+import promise from 'eslint-plugin-promise'
+import babelParser from '@babel/eslint-parser'
+import importplug from 'eslint-plugin-import'
+// eslint-disable-next-line import/extensions
 import eslintPluginPrettierRecommended from 'eslint-plugin-prettier/recommended'
-import globals from "globals";
+import globals from 'globals'
 
 export default [
   js.configs.recommended,
   eslintPluginPrettierRecommended,
   {
     plugins: {
-      promise
+      promise,
+      import: importplug
     },
     languageOptions: {
       globals: {
         ...globals.node
-
       },
       parser: babelParser,
       ecmaVersion: 2020,
@@ -24,28 +25,36 @@ export default [
           legacyDecorators: true,
           jsx: true
         },
-        requireConfigFile: false,
+        requireConfigFile: false
       }
     },
     ignores: [
-      "build/*",
-      "dist/*",
-      "node_modules/*",
-      ".snapshots/*",
-      "*.min.js"
+      'build/*',
+      'dist/*',
+      'node_modules/*',
+      '.snapshots/*',
+      '*.min.js'
     ],
     rules: {
-      "space-before-function-paren": 0,
-      "import/export": 0,
-      "promise/catch-or-return": "error",
-      "no-useless-return": 1,
-      "camelcase": 1,
-    },
-  },
-  {
-    files: ["test/*.spec.js"],
-    rules: {
-      "no-undef": 0
+      'space-before-function-paren': 0,
+      'import/export': 0,
+      'promise/catch-or-return': 'error',
+      'no-useless-return': 1,
+      camelcase: 1,
+      'import/extensions': [
+        'error',
+        'always',
+        {
+          js: 'always',
+          jsx: 'always'
+        }
+      ]
     }
   },
-];
+  {
+    files: ['test/*.spec.js'],
+    rules: {
+      'no-undef': 0
+    }
+  }
+]

--- a/transports/http3-quiche/package.json
+++ b/transports/http3-quiche/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@fails-components/webtransport-transport-http3-quiche",
-  "version": "1.3.0",
+  "version": "1.3.1",
   "description": "A component to add webtransport support (server and client) to node.js, transport using libquiche",
   "exports": {
     ".": {

